tests/en/test_rules_en.py

```diff
@@ -491,6 +495,7 @@ def test_potential_pair_she_exclusively_male_name_compound_antecedent_control(se
     def test_potential_pair_person_word_non_capitalized(self):
         self.compare_potential_pair("I saw a job. He was there", 3, False, 5, 0)
 
+    @unittest.skipIf(train_version_mismatch, train_version_mismatch_message)
```

Comment:
This test didn't work with all 3.4.1 models, presumably because it's a biblical name and hence gets a better score now that it used to.